src/chttpd/src/chttpd_view.erl

```diff
@@ -147,12 +147,12 @@ handle_partition_view_req(Req, _Db, _DDoc, _Pk) ->
 
 check_partition_restrictions(#mrargs{} = Args) ->
     Restrictions = [
-        {<<"include_docs">>, Args#mrargs.include_docs, true},
-        {<<"stable">>, Args#mrargs.stable, true},
-        {<<"conflicts">>, Args#mrargs.conflicts, true}
+        {<<"include_docs">>, Args#mrargs.include_docs},
+        {<<"stable">>, Args#mrargs.stable},
+        {<<"conflicts">>, Args#mrargs.conflicts}
     ],
-    lists:foreach(fun ({Param, Field, Value}) ->
-        case Field =:= Value of
+    lists:foreach(fun ({Param, ArgValue}) ->
+        case ArgValue =:= true of
```

Comment:
just say `case ArgValue of` since we match `true` below, and change the `false` below to `_`. Either that or use two clauses and match in the head.